.github/actions/docker-build-backend/action.yml

```
@@ -37,10 +37,14 @@ runs:
         outputs: ./out
         target: ${{ inputs.target }}
 
-    - run: sha256sum out/${{ inputs.name }}
+    - run: sha256sum out/$NAME
```

```suggestion
    - run: sha256sum "out/$NAME"
```
Quote to avoid shell expansion.